Resources/Prototypes/Traits/inconveniences.yml

```diff
@@ -16,3 +16,17 @@
       fourRandomProb: 0
       threeRandomProb: 0
       cutRandomProb: 0
+
+- type: trait
+  id: FrontalLisp
+  name: trait-frontal-lisp-name
+  description: trait-frontal-lisp-desc
+  components:
+    - type: FrontalLisp
+
+- type: trait
+  id: Snoring
+  name: trait-snoring-name
+  description: trait-snoring-desc
+  components:
+    - type: Snoring
```

Comment:
Just ignore `disabilities.yml` and create a new `cm_inconviences.yml` for these two traits in the CM folder, this way we don't have merge conflicts on upstream files

Reply:
got it. keeping the prototypes for FrontalLisp and Snoring in disabilities was causing an error on startup, is it cool if i keep those removed from that file?

Reply:
not a critical error mind you it just got red and angry at me for duplicates

Reply:
Simply changing the ID should work like `CMFrontalLisp` on the CM file should work, do they not show up if you do that?